src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -846,3 +846,14 @@ def validate_artifact_streaming(namespace):
     if namespace.enable_artifact_streaming:
         if hasattr(namespace, 'os_type') and str(namespace.os_type).lower() == "windows":
             raise ArgumentUsageError('--enable-artifact-streaming can only be set for Linux nodepools')
+
+
+def validate_ssh_access(namespace):
+    """Validates the ssh access string."""
+    if namespace.ssh_access is not None:
+        if namespace.ssh_access == '':
+            return
+        if namespace.ssh_access.lower() != "disabled" and \
+                namespace.ssh_access.lower() != "localuser":
+            raise InvalidArgumentValueError(
```

Comment:
As mentioned above, after defining with `arg_type=get_enum_type(["disabled", "localuser"])` in `_params.py`, this part of logic could be removed.

Reply:
Thanks for reviewing! Replied in above thread.

Reply:
Hi @yanzhudd , I have updated code, could you please help take a look? Thanks!